src/kusama-scss/_footer.scss

```diff
@@ -0,0 +1,16 @@
+footer {
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    color: $secondary-darker;
+    font-size: 0.75rem;
+
+    a {
+        color: $secondary-darker;
```

Comment:
In [kusama](https://kusama.network/) i guess the color is set to white for links in the footer.

Reply:
I'll adjust when Iggy is done with the background graphic